.travis.yml

```diff
@@ -77,7 +77,7 @@ script:
   - export CXX="${CXX}-${COMPILER_VERSION}"
   - cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DRC_ENABLE_TESTS=ON -DRC_ENABLE_EXAMPLES=ON -DRC_ENABLE_GTEST=ON -DRC_ENABLE_GMOCK=ON -DRC_ENABLE_BOOST=ON -DRC_ENABLE_BOOST_TEST=ON -G Ninja ..
   - ninja -j1 # Don't run out of memory, please
-  - ctest --output-on-failure
+  - travis_wait ctest --output-on-failure
```

Comment:
Switch from strings to integers in the commands. Profiled and saw that most of the time is spent allocating memory for the strings.